spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala

```diff
@@ -396,6 +401,14 @@ private[spark] class DeltaSharingRestClient(
       }
     }
 
+  private def getUserAgent(): String = {
```

Comment:
Consider adding a comment on why we append streaming version.

Reply:
added